### ISSUE TITLE
Use ubuntu 18.04

### ISSUE THREAD

**Issue by peterbe** [mozilla-services/kinto-dist#224]
Once https://github.com/mozilla-services/kinto-dist/pull/214 lands we should see to use Ubuntu 18.04 in CircleCI. 

See https://blog.ubuntu.com/2018/07/09/minimal-ubuntu-released

> "On Dockerhub, the new Ubuntu 18.04 LTS image is now the new Minimal Ubuntu 18.04 image. Launching a Docker  instance with docker run ubuntu:18.04  therefore launches a Docker instance with the latest Minimal Ubuntu."

**Comment by peterbe** [comment]
Basically, I think this might make the whole CI run slightly smaller. We use docker to build docker containers (with docker-compose!) and ship those to dockerhub. The ones we ship are python-3.6-slim based. The use of Ubuntu 16.04 is only really within CI and it gets thrown away when the tests complete. So storage size doesn't really matter but if the ubuntu docker image is smaller it might build faster due to less bytes but perhaps that kind of stuff is already heavily cached within the CircleCI infra. Being on ubuntu 18.04 might have advantages to the packages we download/need for the sake of starting the real stuff but it's probably minimal. 

In conclusion, the best thing to do is probably to just go ahead and make this change, see if the tests still pass and then just move on. It's always good to not be on aging tech.
